OpenRA.Mods.D2k/UtilityCommands/D2kMapImporter.cs

```diff
@@ -0,0 +1,419 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2014 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using OpenRA.Primitives;
+
+namespace OpenRA.Mods.D2k.UtilityCommands
+{
+	class D2kMapImporter
+	{
+		const int MapCordonWidth = 4;
+
+		readonly Dictionary<int, Pair<string, string>> actorDataByActorCode = new Dictionary<int, Pair<string, string>>
+		{
+			{ 20, Pair.New("wormspawner", "Creeps") },
+			{ 23, Pair.New("mpspawn", "Neutral") },
+			{ 41, Pair.New("spicebloom", "Neutral") },
+			{ 42, Pair.New("spicebloom", "Neutral") },
+			{ 43, Pair.New("spicebloom", "Neutral") },
+			{ 44, Pair.New("spicebloom", "Neutral") },
+			{ 45, Pair.New("spicebloom", "Neutral") },
+
+			// Atreides:
+			{ 4, Pair.New("WALLA", "Atreides") },
+			{ 5, Pair.New("PWRA", "Atreides") },
+			{ 8, Pair.New("CONYARDA", "Atreides") },
+			{ 11, Pair.New("BARRA", "Atreides") },
+			{ 14, Pair.New("REFA", "Atreides") },
+			{ 17, Pair.New("RADARA", "Atreides") },
+			{ 63, Pair.New("LIGHTA", "Atreides") },
+			{ 69, Pair.New("SILOA", "Atreides") },
+			{ 72, Pair.New("HEAVYA", "Atreides") },
+			{ 75, Pair.New("REPAIRA", "Atreides") },
+			{ 78, Pair.New("GUNTOWERA", "Atreides") },
+			{ 120, Pair.New("HIGHTECHA", "Atreides") },
+			{ 123, Pair.New("ROCKETTOWERA", "Atreides") },
+			{ 126, Pair.New("RESEARCHA", "Atreides") },
+			{ 129, Pair.New("STARPORTA", "Atreides") },
+			{ 132, Pair.New("PALACEA", "Atreides") },
+			{ 180, Pair.New("RIFLE", "Atreides") },
+			{ 181, Pair.New("BAZOOKA", "Atreides") },
+			{ 182, Pair.New("FREMEN", "Atreides") },
+			{ 183, Pair.New("SARDAUKAR", "Atreides") },           //// Atreides sardaukar
+			{ 184, Pair.New("ENGINEER", "Atreides") },
+			{ 185, Pair.New("HARVESTER", "Atreides") },
+			{ 186, Pair.New("MCVA", "Atreides") },
+			{ 187, Pair.New("TRIKE", "Atreides") },
+			{ 188, Pair.New("QUAD", "Atreides") },
+			{ 189, Pair.New("COMBATA", "Atreides") },
+			{ 190, Pair.New("MISSILETANK", "Atreides") },
+			{ 191, Pair.New("SIEGETANK", "Atreides") },
+			{ 192, Pair.New("CARRYALLA", "Atreides") },
+			{ 194, Pair.New("SONICTANK", "Atreides") },
+
+			// Harkonnen:
+			{ 204, Pair.New("WALLH", "Harkonnen") },              //// Harkonnen wall
```

Comment:
Yes please. They are pretty redundant. It is not helpful if source code comments describe obvious things.